.vscode/settings.json

```diff
@@ -11,5 +11,14 @@
         "**/.pytest_cache": true,
         "**/chroma.egg-info": true
     },
-    "python.analysis.typeCheckingMode": "basic"
+    "python.analysis.typeCheckingMode": "basic",
+    "python.linting.flake8Enabled": true,
+    "python.linting.enabled": true,
+    // https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8
```

Comment:
The .json spec doesn't permit comments. Is VSCode using a special parser and do we trust that it will always use a special parser?

Reply:
Probably, I'll remove it.